Resources/doc/index.rst

```diff
@@ -0,0 +1,17 @@
+DoctrineBundle
+==============
+
+Doctrine DBAL & ORM Bundle for the Symfony Framework.
+
+Symfony does not force or suggest a specific persistence solution on its users.
+That's why this bundle was removed from the core of the Symfony framework and
```

Comment:
This sentence should probably be removed, because probably only Symfony early adopters will know that this bundle used to be in the Symfony namespace (the renaming in symfony 2.1 is already old history)